_posts/2023-10-24-glossary.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Glossary
+category: [doc]
+tags: [glossary]
+permalink: /glossary/
+
+css:
+  badge: true
+  syntax: true
+  custom: >-
+    table { font-size: .95rem; margin-bottom: 1.5rem; }
+    tr:nth-child(odd) { backgroud-color: #e3edf3; }
+    th, td { padding: .5em; }
+    .box { border: 1px solid #e3edf3; padding: 1rem; }
+    .plaintext { color: mediumseagreen; }
+
+excerpt: Words and symbols used in peerannot
+featured: true
+---
+
+## Symbols
+
+| Name                    | Definition                                                            | Mathematical Definition                                                                                                                              |
+| ----------------------- | --------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| $n_{task}$              | The total number of tasks in a dataset                                |                                                                                                                                                      |
+| $n_{worker}$            | The total number of workers in a dataset                              |                                                                                                                                                      |
+| $\mathcal{A(x\_i)}$     | The set of workers that answered the task $i$                         | $\\{j\in[ n_{worker} : w_j \text{ answered } x_i]\\}$                                                                                                |
+| $\mathcal{T(w\_j)}$     | The set of tasks answered by the worker $j$                           | $\\{i\in[ n_{task} : w_j \text{ answered }x_i]\\}$                                                                                                   |
+| $\mathcal{Lab(x\_i)}$   | The vector of answered labels of the task $i$                         | $(y_i^{(j)})_{j\in\mathcal{A(x\_i)}}$                                                                                                                |
+| $y_i^*$                 | The true label of the task $i$                                        |                                                                                                                                                      |
+| $ŷ_i^{agg}$             | The computed label of the task $i$ given the aggregation $agg$ method |                                                                                                                                                      |
+| $\pi^{(j)}$             | The confusion matrix of the worker $j$                                |                                                                                                                                                      |
```

Comment:
Mathematical definition: \pi^{(j)}_{k,\ell}=\mathbb{P}(y_i^{(j)​}=\ell∣y_i^\star​=k) \forall (\ell,k)\in [K]^2